pkg/traceability/traceability.go

```diff
@@ -211,3 +230,78 @@ func updateEvent(batch publisher.Batch, events []publisher.Event) {
 	realPtrToEvents := (*[]publisher.Event)(ptrToEvents)
 	*realPtrToEvents = events
 }
+
+func registerHealthCheckers(config *Config) {
+	// register a unique healthchecker for each potential host
+	for i := range config.Hosts {
+		ta := &traceabilityAgentHealthChecker{
+			protocol: config.Protocol,
+			host:     config.Hosts[i],
+			proxyURL: config.Proxy.URL,
+			timeout:  config.Timeout,
+		}
+
+		hcJob := &condorHealthCheckJob{
+			agentHealthChecker: ta,
+		}
+		jobs.RegisterIntervalJob(hcJob, config.Timeout)
```

Comment:
This can return an error.  You probably should check for that

Reply:
I can and will. But if I dig into RegisterIntervalJob, it looks like it never really returns anything except nil.